contracts/SSVNetwork.sol

```diff
@@ -100,56 +157,121 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         emit OperatorRemoved(operatorId);
     }
 
-    function updateOperatorFee(uint64 operatorId, uint64 fee) external {
+    function declareOperatorFee(uint64 operatorId, uint256 fee) onlyOperatorOwnerOrContractOwner(operatorId) external {
         Operator memory operator = _operators[operatorId];
-        if (operator.owner != msg.sender) revert CallerNotOwner();
 
-        _operators[operatorId] = _setFee(operator, fee);
+        if (fee < MINIMAL_OPERATOR_FEE) {
+            revert FeeTooLow();
+        }
+
+        if (fee.shrink() > operator.fee * (10000 + _operatorMaxFeeIncrease) / 10000) {
```

Comment:
Please improve the equation, its not clear the 10000's, also add a comment on calculation